app/src/lib/stores/app-store.ts

```diff
@@ -2196,7 +2199,8 @@ export class AppStore extends TypedBaseStore<IAppState> {
 
     const diff = await getWorkingDirectoryDiff(
       repository,
-      selectedFileBeforeLoad
+      selectedFileBeforeLoad,
+      this.hideWhitespaceInDiff
```

Comment:
I think this should also be gated behind the feature flag, right?
```suggestion
      enableHideWhitespaceInDiffOption() && this.hideWhitespaceInDiff
```